tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/controllers/EndringsmeldingNavController.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.amt.tiltak.deltaker.controllers
+
+import no.nav.amt.tiltak.deltaker.service.EndringsmeldingService
+import org.apache.commons.lang3.NotImplementedException
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.*
+
+@RestController
+@RequestMapping(value = ["/api/nav-ansatt/endringsmelding"])
+class EndringsmeldingNavController(
+	private val service: EndringsmeldingService
+) {
+
+	@GetMapping("/gjennomforing/{gjennomforingId}")
```

Comment:
Pathen for dette i den andre kontrolleren er `/deltaker/{deltakerId}/startdato`.
Det fremstår som det er forskjellige ressurser som man spør etter.

Kan hende at pathen på begge kunne vært `/api/nav-ansatt/endringsmelding/start-dato/...`

Reply:
Det er to helt forskjellige ting, egentlig også helt forskjellige oppgaver å løse men starta på det for å lettere se hvordan man skulle få tak på meldingene etterpå.. Denne henter alle endringsmeldinger på en gjennomføring i motsetning til å lage endringsmelding på deltaker. Tanken er at det ikke nødvendigvis er en endringsmelding av typen oppstartsmelding.

Reply:
Selv om man ønsker å hente forskjellige typer endringsmeldinger fra samme endepunkt så burde ikke /gjennomforing ligge i pathen, det holder at man bruker et query parameter med gjennomforingId